docs/development/style-guide.rst

```diff
@@ -0,0 +1,237 @@
+********************************************************
+Astropy Style Guide: A Writing Resource for Contributors
+********************************************************
+
+Abbreviations
+=============
+
+Place abbreviations such as i.e. and e.g. within parentheses, where they are
+followed by a comma. Alternatively, consider using "that is" and “for example”
```

Comment:
💯  It always annoys me that people don't use comma, so I am glad this is written down now!